test/unit/auth-interface.test.js

```diff
@@ -90,6 +91,23 @@ describe('AuthInterface', function () {
         expect(authInterface.get('nonce')).to.be(NONCE);
     });
 
+    it('should update user parameters with falsy value', function () {
+        var fakeAuthObj,
+            fakeAuth,
+            authInterface,
+            valuesToTestWith = ['', null, undefined, false, 0];
```

Comment:
@harryi3t are you sure you want to override `false` or 0?

Or should we `value === '' || _.isNil(value)` ?